host/src/pack/dpkg/opt/cfc/mwc/bin/lg_launcher.sh

```diff
@@ -8,10 +8,14 @@ then
     exit 1
 fi
 
+# Resume CiV
+/opt/cfc/mwc/bin/resume_civ.sh
+
 num_start_app=`ps aux | grep startapp | grep -v grep | wc -l`
 
 if [ "$num_start_app" -lt "1" ];
 then
+    /opt/cfc/mwc/bin/resume_civ.sh
```

Comment:
partner feedback the change introduce boot time impact. may need optimize to reduce redundant check.